health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/approvePopUp.js

```
@@ -0,0 +1,104 @@
+            onSubmit(comment);  // Pass the comment via onSubmit prop
+        }
+
+        // Close the popup after submitting the comment
+        onClose();
+    };
+
+    return (
+        <PopUp
+            onClose={onClose}
+            heading={t(`HCM_MICROPLAN_SEND_FOR_APPROVAL_LABEL`)}
+            children={[
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)